cfy_manager/main.py

```diff
@@ -484,6 +487,64 @@ def db_shell(**kwargs):
             'DB shell is only accessible with the installed manager config.')
 
 
+@argh.named('fetch')
+@config_arg
+@argh.decorators.arg('-v', '--verbose',help=VERBOSE_HELP_MSG, default=False)
+@argh.decorators.arg('-s', '--skip-db', help=LOGS_SKIP_DB_HELP_MSG,
+                     default=False)
+def logs_fetch(**kwargs):
+    """Download logs from all cloudify managers/dbs/brokers."""
+    setup_console_logger(verbose=kwargs['verbose'])
+    config.load_config(kwargs.get('config_file'))
+    if service_is_configured(MANAGER_SERVICE):
+        if is_all_in_one_manager():
+            # We could log an error, but since we add the monitoring service
+            # by default, we might as well just allow it.
+            nodes = [config[MANAGER][PRIVATE_IP]]
```

Comment:
```suggestion
            nodes = {config[MANAGER][PRIVATE_IP]}
```
wanna make it a set? it's a set in the other branch of the if

Reply:
Sure, can do.